### PR TITLE
Node: Added missing windowsHide properties to SpawnOptions

### PR DESCRIPTION
**Pull request by BrunoScheufler**
Added missing `windowsHide` property to SpawnOptions interface of the child_process module

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: [child_process SpawnOptions](https://nodejs.org/dist/latest-v9.x/docs/api/child_process.html#child_process_child_process_spawn_command_args_options)
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Review by Flarna — APPROVED** [review]

**Comment by Flarna** [comment]
@BrunoScheufler Unrelated CI fails are usually fixed within a few days. You have following options in such cases:
- Wait until unrelated CI issues are fixed (or create PRs to fix it) and then retrigger CI by merge master, push a commit or close/open your PR
- Mention maintainers (e.g. andy-ms, RyanCavanaugh,..) of this repo (not the module reviewers, they have no rights) once PR is approved that CI fail is unrelated.

**Comment by Flarna** [comment]
@BrunoScheufler maybe it's already solved. Just try to reopen the PR.

**Comment by BrunoScheufler** [comment]
Will do!

**Pull request closed without merging**
